composer.json

```diff
@@ -10,7 +10,7 @@
     }
   ],
   "require": {
-    "php": ">=5.4"
+    "php": ">=5.6"
```

Comment:
Constant array properties are a feature of PHP 5.6. We need at least PHP 5.6 to be able to parse the code.

Reply:
I think it's safe to drop php5.4 in 2021 ^^

But we need to update the module compatibility range to >= 1.7.4
because php < 5.6 was dropped starting from 1.7.4 https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart

Reply:
I just did it https://github.com/PrestaShop/ps_socialfollow/pull/26/commits/a7d201c3102fb9b217825a5a494d92cbbad52f38